legend-engine-xt-persistence-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/persistence/test/assertion/ActiveRowsEquivalentToJson.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.persistence.test.assertion;
+
+import org.finos.legend.engine.protocol.pure.v1.model.data.ExternalFormatData;
+import org.finos.legend.engine.protocol.pure.v1.model.test.assertion.TestAssertion;
+
+public class ActiveRowsEquivalentToJson extends TestAssertion
```

Comment:
This seems bit off we are comparing rows to json, I would rather prefer rows to csv
Also we are using inheritance for Active/Non-Active. Do you think using composing would be better?

Reply:
As discussed in email, we will add in AllRowsEquivalentToJson and ActiveRowsEquivalentToJson for now, and subsequently add in AllRowsEquivalentToCSV and ActiveRowsEquivalentToCSV when the support for CSV format is in place. In other words, eventually we will support four modes: all/active assertion in Json/CSV format